projects/opentrain/gtfs_pipeline/heatmap/HeatMap.java

```diff
@@ -0,0 +1,549 @@
+      graphics2D.setFont(new Font("TimesRoman", Font.BOLD, 20));
+      graphics2D.drawString("X", stopAround.i - 1, stopAround.j - 1);
+    }
+
+    graphics2D.dispose();
+    // Save file
+    boolean createImage = false;
+    try {
+      createImage = ImageIO.write(image, "png", new File(heatMapPath.get()));
+
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    if (!createImage) {
+      System.out.println("can't create image");
+    }
+  }
+
+  // Functions by different data and sites to get x pixel on the map:
+  // X = 31.84931551 x1^2 - 155.5785894 x1 x2 + 134.2697945 x2^2 + 3427.951862 x1 - 3846.07897 x2 +
+  // 2512.552383
+  public int getXpixel(double longitude, double latitude) {
+    int xPixel =
+        (int)
+            ((31.84931551 * (Math.pow(longitude, 2)))
+                + (-155.5785894 * longitude * latitude)
+                + (134.2697945 * (Math.pow(latitude, 2)))
+                + (3427.951862 * longitude)
+                - (3846.07897 * latitude)
+                + 2512.552383);
+    /*#2(int)( -15980 + (69.73)*longitude + (405.4)*latitude);*/
+    /*#1
+    (int)
+            (31.84931551 * (Math.pow(longitude, 2))
+                    + -155.5785894 * longitude * latitude
+                    + 134.2697945 * (Math.pow(latitude, 2))
+                    + 3427.951862 * longitude
+                    - 3846.07897 * latitude
+                    + 2512.552383);*/
+    return // xPixel;
+    xPixel + 58;
+  }
+
+  // Functions by different data and sites to get y pixel on the map:
+  // Y = 67.87903886 x1^2 - 59.89376472 x1 x2 + 11.68179254 x2^2 - 2895.823625 x1 + 1070.268543 x2 +
+  // 38611.28426
+  public int getYpixel(double longitude, double latitude) {
+    int yPixel =
+        (int)
+            ((67.87903886 * (Math.pow(longitude, 2)))
+                + (-59.89376472 * longitude * latitude)
+                + (11.68179254 * (Math.pow(latitude, 2)))
+                + (-2895.823625 * longitude)
+                + (1070.268543 * latitude)
+                + 38611.28426);
+    /*#2(int)(23100 + (-653.6)*longitude + (-58.28)*latitude);*/
+    /*#1(int)
+    ((67.87903886 * (Math.pow(longitude, 2)))
+            + (-59.89376472 * longitude * latitude)
+            + (11.68179254 * (Math.pow(latitude, 2)))
+            + (-2895.823625 * longitude)
+            + (1070.268543 * latitude)
+            + 38611.28426);*/
+
+    return // yPixel;
+    yPixel + 53;
+  }
+
+  private void arrivalTimeToLocationForMap() {
+    heatMap = new double[770][918];
+
+    // Start stop x,y
+    int y = getYpixel(startStop.getStopLat(), startStop.getStopLon());
+    int x = getXpixel(startStop.getStopLat(), startStop.getStopLon());
+    int startStopX = x;
+    int startStopY = y;
+    heatMap[x][y] = 0;
+
+    for (StopsAround stopAround : stopsAroundList) {
+
+      // Arrival time of each stop from the chosen stop by location
+      y = getYpixel(stopAround.stop.getStopLat(), stopAround.stop.getStopLon());
```

Comment:
It's better to be consistent - first x, then y. So I would switch the line order here.